src/data/pokemon/base_stats.h

```diff
@@ -1669,23 +1669,23 @@ const struct BaseStats gBaseStats[] =
         .bodyColor = BODY_COLOR_BROWN,
         .noFlip = FALSE,
     },
-
+//Galarian form as default
     [SPECIES_MEOWTH] =
     {
```

Comment:
I know you can't wait to add this guy, but this is not the right place to do so. Revert this commit and keep those changes for another branch

Reply:
reverted commit